plugins/github-pull-requests-board/src/api/useGetPullRequestDetails.ts

```diff
@@ -33,6 +33,9 @@ export const useGetPullRequestDetails = () => {
                 id
                 repository {
                   name
+                  owner {
+                    login
```

Comment:
hm won't this be the same as $owner in practice? 🤔

Reply:
Yeah, it's the same

I mostly modified this so that I can use the owner information [here](https://github.com/backstage/backstage/pull/13837/files#diff-8edd789ba2c4efc14c1af37160a7a9d344ea1ed77194efcbc91b667ac5544f32R138), without having to forward/pass in additional variables just for the owner.

Alternatively, I can not request the owner in the GraphQL query but manually update the response to set the information, but since this doesn't affect rate limits (I think?), I guess it's just easier to take advantage of Github API? But let me know if you'd rather have me not rely on the Github API for that data

Reply:
Alright, let's keep it like this then.